pkg/sql/schemachanger/testdata/alter_table_add_column

```diff
@@ -510,7 +510,7 @@ upsert descriptor #106
          keyColumnNames:
          - i
   -      name: crdb_internal_index_3_name_placeholder
-  +      name: crdb_internal_index_1_name_placeholder
+  +      name: tbl_pkey
```

Comment:
this is a problem, no? 

Doesn't this mean that at this point, we have two indexes/constraints with this name?